themes/startwords/assets/scss/article/_notes.scss

```diff
@@ -124,8 +124,8 @@ body.article .footnotes {
         }
 
         a {
-            @include highlight-focus;
-            color: $light-purple;
+             @include highlight-focus;
+             color: $light-purple;
```

Comment:
what's up with this added whitespace?

Reply:
I don't know! I removed these and then added them back because I figured out I was changing the styles in the wrong place (not specific enough). I can check and fix.